### PR TITLE
Tests: ABCI connection

### PR DESCRIPTION
**Pull request by Karrenbelt**
## Proposed changes

adding `# pragma no cover` to abci connection tests covered as part of `test_app.py`

## Fixes

If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce? (A **breaking change** is a fix or feature that would cause existing functionality and APIs to not work as expected.)
_Put an `x` in the box that applies_

- [ ] Non-breaking fix (non-breaking change which fixes an issue)
- [ ] Breaking fix (breaking change which fixes an issue)
- [ ] Non-breaking feature (non-breaking change which adds functionality)
- [ ] Breaking feature (breaking change which adds functionality)
- [ ] Refactor (non-breaking change which changes implementation)
- [ ] Messy (mixture of the above - requires explanation!)

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../blob/main/CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `main` branch (left side). Also you should start your branch off our `main`.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have locally run services that could be impacted and they do not present failures derived from my changes

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: tests/test_deployments/test_deployment_code_identical.py [hunk 1]
```diff
@@ -38,13 +38,15 @@
         (connection.TendermintNode, tendermint.TendermintNode),
     ],
 )
-def test_deployment_class_identical(deployment_cls: Type, package_cls: Type) -> None:
+def test_deployment_class_identical(package_cls: Type, deployment_cls: Type) -> None:
```
Comment (Karrenbelt) [inline tests/test_deployments/test_deployment_code_identical.py:41]: switcheroo

**Review by 0xArdi — APPROVED** [review]

File: packages/valory/connections/abci/connection.py [hunk 2]
```diff
@@ -1059,7 +1061,7 @@ def stopped(self) -> bool:
         return self._stop_event.is_set()
 
 
-class TendermintParams:  # pylint: disable=too-few-public-methods
+class TendermintParams:  # pylint: disable=too-few-public-methods  # pragma: no cover (covered via deployments/Dockerfiles/tendermint/tendermint.py)
```
Comment (DavidMinarsch) [inline packages/valory/connections/abci/connection.py:1064]: How do we know the code matches 1:1?

File: tests/test_deployments/test_deployment_code_identical.py (hunk 1 above)
Comment (Karrenbelt) [reply tests/test_deployments/test_deployment_code_identical.py:41]: > How do we know the code matches 1:1?

@DavidMinarsch

Comment (DavidMinarsch) [reply tests/test_deployments/test_deployment_code_identical.py:41]: Ah nice :)

**Review by DavidMinarsch — APPROVED** [review]

**Pull request merged**